### ISSUE TITLE
Missing Images

### ISSUE THREAD

**Issue by ghost** [chrisbenincasa/Swiftopolis#9]
![screen shot 2015-03-14 at 11 04 38 pm](https://cloud.githubusercontent.com/assets/10824143/6654635/86fb7806-ca9e-11e4-9b51-a201a0e7e624.png)

It appears some of your images are not being tracked correctly.

**Comment by chrisbenincasa** [comment]
@Urthstripe29 my apologies. This is due to me commenting out a portion of the script that generates these output files. See: https://github.com/chrisbenincasa/Swiftopolis/blob/master/generate-tiles.sh#L9

I commented this out in the interim because the image generation scripts were running each time I built the project and was making build time quite slow. Everything should work if those lines are uncommented. I will add an issue to track updating the script to make it a bit smarter about generating these files (perhaps only generating them if the `build/` directory isn't present.

**Comment by ghost** [comment]
Yup uncommenting those lines allowed it to build. You might want to remove the group folder from the  Project Navigator. Also you might want the project scheme to default Swiftopolis instead of TileGenerator. Glad to see the map panning is going well.
